scripts/installer-image.yaml

```diff
@@ -0,0 +1,37 @@
+#clear-linux-config
+
+# switch between aliases if you want to install to an actuall block device
+# i.e /dev/sda
+block-devices: [
+   {name: "installer", file: "clear-installer.img"}
+]
+
+targetMedia:
+- name: ${installer}
+  size: "1.2G"
+  type: disk
+  children:
+  - name: ${installer}1
+    fstype: vfat
+    mountpoint: /boot
+    size: "150M"
+    type: part
+  - name: ${installer}2
+    fstype: swap
+    size: "120M"
+    type: part
+  - name: ${installer}3
+    fstype: ext4
+    mountpoint: /
+    size: "958M"
+    type: part
+
+bundles: [os-core, os-core-update, telemetrics, os-installer, openssh-server]
+telemetry: false
+keyboard: us
+language: en_US.UTF-8
+kernel: kernel-native
+
```

Comment:
This config should add:
```
postReboot: false
postArchive: false
autoUpdate: false
```
then the --reboot false not needed on the command line
We do not want to store the log file and the configuration file in the target installer image
We do not want the image to update to the lastest.

Reply:
@dorileo should we drop the '--reboot=false' from the Readme.md?

Reply:
sure

Reply:
done, thanks.